.github/workflows/ci.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: |
           docker network create --driver bridge delphi-net
           docker run --rm -d -p 13306:3306 --network delphi-net --name delphi_database_epidata --cap-add=sys_nice delphi_database_epidata
-          docker run --rm -d -p 10080:80 --env "SQLALCHEMY_DATABASE_URI=mysql+mysqldb://user:pass@delphi_database_epidata:3306/epidata" --env "FLASK_SECRET=abc" --env "FLASK_PREFIX=/epidata" --network delphi-net --name delphi_web_epidata delphi_web_epidata
+          docker run --rm -d -p 10080:80 --env "MODULE_NAME=delphi.epidata.server.main" --env "SQLALCHEMY_DATABASE_URI=mysql+mysqldb://user:pass@delphi_database_epidata:3306/epidata" --env "FLASK_SECRET=abc" --env "FLASK_PREFIX=/epidata" --network delphi-net --name delphi_web_epidata delphi_web_epidata
```

Comment:
question: where is `MODULE_NAME` used?

Reply:
its used in the base image that gets built upon in `devops/Dockerfile`, "`tiangolo/meinheld-gunicorn`".  see [the documentation](https://github.com/tiangolo/meinheld-gunicorn-docker#module_name) for that image, which i also [mentioned](https://github.com/cmu-delphi/delphi-epidata/issues/1037#issuecomment-1416172115) in the associated issue (#1037) for this pr.

comments noting the purpose of `MODULE_NAME` along with that documentation link above should probably be included here, in the `Makefile`, and in `devops/Dockerfile`.

`MODULE_NAME` will also need to be populated in automation or ansible or whatever it is that runs the API server code, which we will probably see when we try to test running this on our staging and/or qa setups. (cc: @korlaxxalrok )

Reply:
Yep, looks like we'd need to pass an additional environment variable.